src/script_chainer/win_exe/script_runner.py

```
@@ -412,43 +504,74 @@ def run_chain(chain_name: str = '01', shutdown_delay: int = 0) -> None:
                         )
 
-                # 定时推送日志
-                log_notifier: LogNotifier | None = None
-                if ctx is not None and script_config.notify_log_interval > 0:
-                    log_notifier = LogNotifier(
-                        ctx=ctx,
-                        title=f'{ctx.notify_config.title} - {script_config.script_display_name} 日志',
-                        interval=script_config.notify_log_interval,
-                    )
-                    log_notifier.start()
+                # 判断是否挂靠到上一个脚本（复用其 log_notifier）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)